frontend/apps/pwa/src/components/formFields/CheckboxFieldEditor.tsx

```diff
@@ -31,7 +30,6 @@ export const CheckboxFieldEditor: React.FC<FieldEditorProps> = ({
           {field.name}
         </label>
       </div>
-      <FieldDescription text={field.description} />
```

Comment:
Why was this removed?

Reply:
had to do that to reduce the amount of duplicated code for sonar